src/game.zig

```diff
@@ -941,6 +947,8 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 		const fricMul = speedMultiplier*baseFrictionCoefficient*if(Player.isFlying.load(.monotonic)) 1.0 else mobility;
 
 		const forward = vec.rotateZ(Vec3d{0, 1, 0}, -camera.rotation[2]);
+		const lerpAmount: Vec3d = @splat(std.math.clamp(density/@max(1.0, maxDensity), 0.0, 1.0));
+		const lerpedDir = std.math.lerp(forward, camera.direction, lerpAmount);
```

Comment:
`lerpedDir` is not a particularly good variable name in my opinion.
I'd suggest to keep this named `forward` and rename the variable above instead.